cmd/captain/run.go

```diff
@@ -325,7 +325,7 @@ func bindRunCmdFlags(cfg Config, cliArgs CliArgs) Config {
 		}
 
 		if len(cliArgs.reporters) > 0 {
-			reporterConfig := make(map[string]string)
```

Comment:
this could also warrant a test? This kind of behavior can be a little finnicky

Reply:
See https://github.com/rwx-research/captain/pull/16#discussion_r1160715650 about the challenges of testing the reporter output 😐

Reply:
In short, yes we should test this but exactly how I'm not quite sure yet since we want to run the old integration tests in a matrix and reporter output is subject to change (e.g. JSON/XML in non-breaking ways and the markdown if we want it to look different)